pkg/proc/threads.go

```diff
@@ -256,6 +252,20 @@ func next(dbp Process, stepInto, inlinedStepOut bool) error {
 		return err
 	}
 
+	if backward {
+		// Insure that pcs contains firstPCAfterPrologue when reverse stepping.
+		found := false
+		for _, pc := range pcs {
+			if pc == firstPCAfterPrologue {
+				found = true
+				break
+			}
+		}
+		if !found {
```

Comment:
AllPCsBetween returns all addresses that have the is_stmt flag set, 'firstPCAfterPrologue' might not be there (especially if we detected it using disassembly instead of getting it from debug_line).

Reply:
Ack, that's right.